src/_tutorials/web/images/portmanteaux-code.svg

```diff
@@ -0,0 +1,3 @@
: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 600px; margin-left: 127px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 16px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: none; white-space: nowrap; ">String containing URI</div></div></div></foreignObject><text x="127" y="605" fill="#000000" font-family="Helvetica" font-size="16px" text-anchor="middle">String containing URI</text></switch></g><rect x="42" y="620" width="370" height="20" fill="#ffefaa" stroke="none" pointer-events="none"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 630px; margin-left: 227px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 16px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: none; white-space: nowrap; ">Callback function, called when request completes</div></div></div></foreignObject><text x="227" y="635" fill="#000000" font-family="Helvetica" font-size="16px" text-anchor="middle">Callback function, called when request complet...</text></switch></g><path d="M 41.85 540.22 L 17 540.06 Q 7 540 7 530 L 7 353 Q 7 343 17 343 L 99.63 343" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 104.88 343 L 97.88 346.5 L 99.63 343 L 97.88 339.5 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 212.2 559.16 L 212 394.12" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 212 391.12 L 214.01 395.12 L 212 394.12 L 210.01 395.12 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 213.53 600.6 L 247 600.14 Q 257 600 257 590 L 257 394.96" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 257 391.96 L 259 395.96 L 257 394.96 L 255 395.96 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 412 629.66 L 437 629.9 Q 447 630 447 620 L 447 415.62" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 447 411.12 L 450 417.12 L 447 415.62 L 444 417.12 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Could you update the code to match the updated tutorial?

```dart
Future<void> makeRequest(Event _) async {
  const path = 'https://dart.dev/f/portmanteaux.json';
  final httpRequest = HttpRequest();
  httpRequest
    ..open('GET', path)
    ..onLoadEnd.listen((e) => requestComplete(httpRequest))
    ..send('');
}
```

Also:

1. When I'm viewing it, it seems the code block is hanging outside the background. Feel free to make the background larger to get it to fit.
2. As for the Portmanteaux application example, I think it's fine in this case that the application image is not an actual image for size purposes. But could you update the button text to say "Get portmanteaux" and perhaps use a rounded rectangle for the button to match the current UI(I believe Diagrams.net has this as a built in shape, but if not the rectangle is fine).

DartPad rendered version for reference:
![image](https://user-images.githubusercontent.com/18372958/121575355-e49ea880-c9ec-11eb-851b-91d10e46a6b7.png)